src/Microsoft.PowerShell.Commands.Management/commands/management/Hotfix.cs

```diff
@@ -142,15 +134,6 @@ protected override void BeginProcessing()
                         catch (SystemException) // thrown by SecurityIdentifier.constr
                         {
                         }
-                        // catch (ArgumentException) // thrown (indirectly) by SecurityIdentifier.constr (on XP only?)
-                        // { catch not needed - this is already caught as SystemException
-                        // }
-                        // catch (PlatformNotSupportedException) // thrown (indirectly) by SecurityIdentifier.Translate (on Win95 only?)
```

Comment:
Wow, was powershell v1 supported on win95 (I though PS came out only around the XP times)? Or is this still from Monad days?

Reply:
Not sure if v1 actually supported 95, although I suspect not.  This error seems like a developer trying to cover their bases for exceptions.